pyproject.toml

```diff
@@ -77,6 +77,7 @@ polars = { version = ">=0.19.3,<1", optional = true }
 psycopg2 = { version = ">=2.8.4,<3", optional = true }
 pydata-google-auth = { version = ">=1.4.0,<2", optional = true }
 pydruid = { version = ">=0.6.5,<1", optional = true }
+pyexasol = { version = ">=0.25.2,<1", optional = true, extras = ["pandas"] }
```

Comment:
Not 100% convinced we should bring in this extra. There's a pretty sizable hit from all the CSV conversion (despite being a thing that ostensibly helps performance).

Reply:
I removed the hit from `export_to_pandas` (by doing the conversion ourselves).

I'm going to keep the extra for use with `import_from_pandas` which simplifies inserting into in-memory tables.